<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>510</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Plot selection</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="layout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="accessibleName">
      <string>tabWindow</string>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="rawTab">
      <property name="accessibleName">
       <string>Raw</string>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <attribute name="title">
       <string>Raw</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </widget>
     <widget class="QWidget" name="averagePowerBandTab">
      <property name="accessibleName">
       <string>Average Band Power</string>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <attribute name="title">
       <string>Average Band Power</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3"/>
     </widget>
     <widget class="QWidget" name="fftTab">
      <attribute name="title">
       <string>FFT</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6"/>
     </widget>
     <widget class="QWidget" name="oneChannelTab">
      <property name="accessibleName">
       <string>One Channel Features</string>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <attribute name="title">
       <string>One Channel Features</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="title">
          <string>Features</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QCheckBox" name="hfdCB">
            <property name="text">
             <string>Higuchi Fractal Dimension</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="pfdCB">
            <property name="text">
             <string>Petrosian Fractal Dimension</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="hjorthActivityCB">
            <property name="text">
             <string>Hjorth Activity</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="hjorthMobilityCB">
            <property name="text">
             <string>Hjorth Mobility</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="hjorthComplexityCB">
            <property name="text">
             <string>Hjorth Complexity</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="mseCB">
            <property name="text">
             <string>Sample Entropy</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="lzcCB">
            <property name="text">
             <string>Lempel-Ziv Complexity</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="dfaCB">
            <property name="text">
             <string>Detrended Fluctuation Analysis</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="twoChannelsTab">
      <property name="accessibleName">
       <string>Two Channels Features</string>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <attribute name="title">
       <string>Two Channels Features</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Features</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QCheckBox" name="cccCB">
            <property name="text">
             <string>Cross Correlation Coeficient</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="slCB">
            <property name="text">
             <string>Synchronization Likelihood</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Channeless Features</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Features</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="engagementCB">
            <property name="toolTip">
             <string>This feature doesn't use an specific channel</string>
            </property>
            <property name="text">
             <string>Engagement Level</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="nameTB">
     <property name="toolTip">
      <string>Window name</string>
     </property>
     <property name="statusTip">
      <string>Window name</string>
     </property>
     <property name="whatsThis">
      <string>Window name</string>
     </property>
     <property name="placeholderText">
      <string>Window name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="apButton">
     <property name="text">
      <string>Add plot</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
